calblueprint/ogsc
# Generic Player Dashboard

# Add Linked Players Dashboard

- add star indicator for linked players on player dashboard
- added a separate "Your Players" tab on the player dashboard
- non-admin users have full access to their linked players' player profiles
- add the appropriate access permissions inside lib/access (including absences)

## How to Test/Use PR feature

1) log in as a mentor (easiest role to test because they should already have linked players)
2) go to localhost:[]/[role]/players -should be able to see the star indicator and separate tab for your linked players
3) from there, click on a linked player and you should be able to see their full player profile & if you click on an unlinked player, you should only be able to see the "Overview" and "Highlights" pages

## PR Checklist

Does your branch (check if true):
[X] work when you run `yarn build`?
[X] successfully run `yarn:db-migrate up` without yielding any errors?
[X] successfully run `yarn prisma introspect` without yielding any errors?
[X] successfully run `yarn dev` and support all changes that your branch intends to perform?

## Migrations

N/A

## Screenshots
Generic Player Dashboard:
![image](https://user-images.githubusercontent.com/65790318/110242924-07608300-7f0d-11eb-9362-08cfe25bec0d.png)

Linked Players View of Dashboard:
![image](https://user-images.githubusercontent.com/65790318/110242938-19422600-7f0d-11eb-9dad-bf4107e6d373.png)

Linked Player Profile View:
![image](https://user-images.githubusercontent.com/65790318/110242949-265f1500-7f0d-11eb-8f5a-9e901f53543b.png)

Unlinked Player Profile View:
![image](https://user-images.githubusercontent.com/65790318/110242981-48589780-7f0d-11eb-971c-c387fe93a7a3.png)



CC: @sydbui
